app/models/user.rb

```diff
@@ -14,6 +18,7 @@ class User < ActiveRecord::Base
     message: I18n.t("validations.accepted"),
   }
   validates :password, presence: true, on: :create
+  validates :phone_number, phony_plausible: { normalized_country_code: "US" }
```

Comment:
Do you need `allow_nil` or `allow_blank` here? What's the expectation for missing phone numbers?